cuda_bindings/docs/source/conf.py

```diff
@@ -85,6 +92,16 @@
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ["_static"]
 
+# skip cmdline prompts
+copybutton_exclude = ".linenos, .gp"
+
+intersphinx_mapping = {
+    "python": ("https://docs.python.org/3/", None),
+    "numpy": ("https://numpy.org/doc/stable/", None),
+    "nvvm": ("https://docs.nvidia.com/cuda/libnvvm-api/", None),
+    "nvjitlink": ("https://docs.nvidia.com/cuda/nvjitlink/", None),
```

Comment:
Any reason to add nvrtc or the CUDA runtime / driver APIs here?

Reply:
Currently the two codegens set different expectations regarding API doc generation. The codegen used by driver/runtime/nvrtc regenerates the entire C API references in the docs (with signatures adjusted to match Python(, whereas the codegen used by nvvm/nvjitlink generates basic docs with a `see also` link to the corresponding C API. The addition of these links here contain proper `objects.inv` that allows cross-linking, e.g.
https://nvidia.github.io/cuda-python/pr-preview/pr-547/cuda-bindings/latest/module/nvjitlink.html#cuda.bindings.nvjitlink.create
The see also link works.

Reply:
(FWIW regarding `objects.inv`, it's a binary that is generated by Sphinx and can be i[ntrospected by the intersphinx plugin](https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html#showing-all-links-of-an-intersphinx-mapping-file).)